docs/xls-66d-lending-protocol/reference/transactions/loanset.md

```diff
@@ -0,0 +1,108 @@
+| `LoanBrokerID`         | String    | Hash256      | Yes       | The ID of the `LoanBroker` ledger entry. |
+| `Flags`                | String    | UInt32       | No        | Flags for the loan. |
+| `Data`                 | String    | Blob         | No        | Arbitrary metadata in hex format (max 256 bytes). |
+| `Borrower`             | String    | AccountID    | Yes       | The account address of the borrower. |
+| `LoanOriginationFee`   | Number    | Number       | No        | The fee paid to the `LoanBroker` owner when the loan is created. |
+| `LoanServiceFee`       | Number    | Number       | No        | The fee paid to the `LoanBroker` owner with each loan payment. |
+| `LatePaymentFee`       | Number    | Number       | No        | The fee paid to the `LoanBroker` owner for late payments. |
+| `FullPaymentFee`       | Number    | Number       | No        | The fee paid to the `LoanBroker` owner for early full repayment. |
+| `InterestRate`         | Number    | UInt16       | No        | The annualized interest rate of the loan in basis points. |
+| `LateInterestRate`     | Number    | UInt16       | No        | The premium (in basis points) added to the interest rate for late payments. Valid values range from `0` to `10000` (0% - 100%). |
+| `ClosingInterestRate`  | Number    | UInt16       | No        | The fee charged for making early loan repayments (in bps). Valid values range from `0` to `100000` (0% - 100%). |
+| `PrincipalRequested`   | Number    | Number       | Yes       | The principal loan amount requested by the borrower. |
+| `StartDate`            | Number    | UInt32       | Yes       | The timestamp of when the loan started, in [seconds since the Ripple Epoch][]. |
+| `PaymentsTotal`        | Number    | UInt32       | No        | The total number of payments to be made against the loan. |
+| `PaymentInterval`      | Number    | UInt32       | No        | The number of seconds between loan payments. |
+| `GracePeriod`          | Number    | UInt32       | No        | The number of seconds after the loan's payment due date when it can be defaulted. |
+| `Lender`               | Object    | STObject     | Yes       | An inneer object that contains the signature of the lender. |
+
+### Lender Fields
+
+An inner object that contains the signatures of the lender of the transaction. The object contains the following fields:
+
+| Field Name           | JSON Type | Internal Type | Required? | Description |
+|:-------------------- |:----------|:-------------|:----------|:------------|
+| `SigningPubKey` | String | STBlob | Yes | The public key used to verify the validity of the signature. |
+| `Signature` | String | STBlob | Yes | The signature over all signing fields, including the `Signature` of the borrower. |
+| `Signers` | List | STArray | An array of transaction signatures from the owners of the `LoanBroker` ledger entry. |
+
+The final transaction must include either `Signature` or `Signers`.
+
```

Comment:
Flags have been added for this transaction:

https://github.com/XRPLF/XRPL-Standards/blob/81e2ff8954763fa572ba0bb937333391327dca63/XLS-0066d-lending-protocol/README.md#3211-flags